core/export.py

```diff
@@ -0,0 +1,192 @@
+                        inner_border = df_to_tuple(int_border_df)
+                        polygon_borders.append(inner_border)
+                all_border_df = pd.concat([all_border_df, border_df])
+                multi_polygons.append(polygon_borders)
+
+            new_border_df = all_border_df
+            multi_polygon = MultiPolygon(multi_polygons)
+            new_feature = Feature(geometry=multi_polygon,
+                                  properties={"id": str(countryCode),
+                                              "polygon-id": str(countryCode) + '-' + str(polygon_id),
+                                              "count": current})
+
+        borders_df = pd.concat([borders_df, new_border_df])
+        feature_list.append(new_feature)
+
+    borders_df.to_csv(border_filename, index=False)
+    feature_collection = FeatureCollection(feature_list)
+
+    with open(geo_filename, 'w', encoding='utf8') as geojson_file:
+        dump(feature_collection, geojson_file)
+
+
+def geo_to_projection(geo_filename, projected_geo_filename):
+
+    # Using Javascript libraries instead of python bundles.
+    os.system(
+        "npx geoproject 'd3.geoNaturalEarth1().fitSize([1250, 750], d)' < " + geo_filename + " > " + projected_geo_filename)
+
+
+def geo_to_topo(projected_geo_filename, topo_filename):
+
+    # Using Javascript libraries instead of python bundles.
+    os.system("npx geo2topo tiles=" + projected_geo_filename + " \
+                | npx toposimplify -p 0.00005 \
+                | npx topoquantize 1e9 > " + topo_filename)
+
+
+def create_polygon(row, radius):
+    diameter = radius * 2
+    return shpPolygon([(row['X'], row['Y']),
+                       (row['X'] + diameter, row['Y']),
+                       (row['X'] + diameter, row['Y'] + diameter),
+                       (row['X'], row['Y'] + diameter)])
+
+
+def generate_plot(cell_filename, border_filename):
+    borders = pd.read_csv(border_filename)
+    cells = pd.read_csv(cell_filename)
+
+    fig = plt.figure()
+    ax = fig.add_subplot(111, aspect='equal')
+    plt.xlim([0, max(cells["X"] + 1)])
+    plt.ylim([0, max(cells["Y"] + 1)])
+    n = cells.shape[0]
+    patches = []
+
+    for i in range(0, n):
+        patches.append(matplotlib.patches.Rectangle(
+            (cells.loc[i, "X"] + .5, cells.loc[i, "Y"] + .5), 0.2, 0.2, color="#111111"))
+    ax.add_collection(PatchCollection(patches, alpha=0.1))
+
+    for p in np.unique(borders["PolygonID"]):
+        ax.plot(borders.loc[borders["PolygonID"] == p, "X"],
+                borders.loc[borders["PolygonID"] == p, "Y"])
+    plt.show()
+
+
+def df_to_tuple(border_df):
+    border = tuple((zip(border_df.X / 1000, border_df.Y / 1000)))
+    return border
+
+
+def main():
+    cells = '../data/output/v1/cells.csv'
+    borders = '../data/output/v1/borders.csv'
+    geo = '../data/output/v1/geo.json'
+    projected_geo = '../data/output/v1/projected_geo.json'
+    topo = '../data/output/v1/topo.json'
```

Comment:
This will have to be parameterized for the node script but is sufficient for now 👍